.github/workflows/schedule.yml

```diff
@@ -324,4 +324,4 @@ jobs:
           printf "[INFO] Notify ID: '%s'\n" "${{ env.NOTIFY_ID }}"
           printf "[INFO] Software Name: '%s'\n" "${{ env.SOFTWARE_NAME }}"
           printf "[INFO] Software Version: '%s'\n" "${{ env.SOFTWARE_VERSION }}"
-          ve1/bin/pytest tests/functional/step_defs/test_submitted_charts.py --log-cli-level=WARNING --tb=short
+          ve1/bin/pytest -n 4 tests/functional/step_defs/test_submitted_charts.py --log-cli-level=WARNING --tb=short
```

Comment:
Would be good to set 4 as an environment variable so it is set in just one place. Then if we ever want to try different numbers in needs just a single edit. Guess a counter to that would be if see a future need to use different number for different pytest invocations.

Reply:
Agreed. I realized we don't need to support parallelization for submitted chart testing because there's only one pytest file with one test case. After removing, there was only one ocurrence of this value n=4. Though I strongly agree we should deduplicate as much as possible.